python/cdp/evm_client.py

```diff
@@ -555,7 +554,7 @@ async def update_account(
         """
         account = await self.api_clients.evm_accounts.update_evm_account(
             address=address,
-            update_evm_account_request=UpdateEvmAccountRequest(
+            create_evm_account_request=CreateEvmAccountRequest(
```

Comment:
weird that although `UpdateEvmAccountRequest` exists, the update_evm_account is expecting a `CreateEvmAccountRequest`. maybe sthg about them having the same structure so openapi just uses `CreateEvmAccountRequest` for both create and update